tests/Responses/Expected/checkout/reselect_shipping_response.json

```diff
@@ -0,0 +1,19 @@
+{
+    "id": @integer@,
+    "items": @array@,
```

Comment:
I'm not sure about mocking so many things... Cannot we just add basic data such as items or items total to these responses?